Makefile

```diff
@@ -308,7 +308,17 @@ sentry-stop:
 build-slate:
 	bash scripts/slate.sh
 
+build-contract-tests-hooks:
+ifeq ($(OS),Windows_NT)
+	go build -mod=readonly $(BUILD_FLAGS) -o build/contract_tests.exe ./cmd/contract_tests
+else
+	go build -mod=readonly $(BUILD_FLAGS) -o build/contract_tests ./cmd/contract_tests
+endif
+
+contract-tests: build build-contract-tests-hooks
+	dredd
```

Comment:
what's this? should we install it during `get_tools` https://github.com/tendermint/tendermint/blob/master/scripts/get_tools.sh ? if not, then there should be a link to it at least

Reply:
also github.com/snikch/goodman needs to be installed via `get_tools` ^

Reply:
`dredd` is the tool that runs every endpoint he finds in the swagger file and check if the payload of the calls agree with the documented one

Reply:
Right! I added only in the get_node.sh but I forgot to describe it here for local usage, will do it now

Reply:
Just added some documentation under `CONTRIBUTING`